streams/src/main/java/org/apache/kafka/streams/StreamsMetrics.java

```diff
@@ -31,84 +31,108 @@
 public interface StreamsMetrics {
 
     /**
-     * Get read-only handle on global metrics registry
+     * Get read-only handle on global metrics registry.
+     *
      * @return Map of all metrics.
      */
     Map<MetricName, ? extends Metric> metrics();
 
     /**
-     * Add a latency and throughput sensor and default associated metrics. Metrics include both latency ones
-     * (average and max latency) and throughput ones (operations/time unit).
+     * Add a latency and throughput sensor and default associated metrics.
```

Comment:
This is not introduced in this PR, but I think we can make this comment more specific and clear. How about:

```
Add a latency and throughput sensor for a specific operation, which will include the following sensor:
1. average latency.
2. max latency.
3. throughput (num.operations / time unit).

Also create a parent sensor with the same metrics that aggregates all entities with the same operation under the same scope if it has not been created.
```